ros_buildfarm/templates/snippet/assigned_node.xml.em

```diff
@@ -0,0 +1 @@
+    <assignedNode>@(node_label if node_label else 'buildslave') || @('_'.join(['', job_type, rosdistro_name, arch])) || @('_'.join(['', job_type, rosdistro_name])) || @('_'.join(['', job_type, arch]))</assignedNode>
```

Comment:
Please wrap this logic into separate lines with individual conditions. In the current form it is extremely difficult to read.